src/ORM/Association/BelongsToMany.php

```diff
@@ -155,9 +155,9 @@ class BelongsToMany extends Association
     /**
      * Order in which target records should be returned
      *
-     * @var mixed
+     * @var \Cake\Database\ExpressionInterface|\Closure|array|string|null
      */
-    protected mixed $_sort = null;
+    protected ExpressionInterface|Closure|array|string|null $_sort = null;
```

Comment:
Can we be more specific about the array value type?

Reply:
You mean in the docblock I guess? I think they array could hold at least `int => string`, `string => string`, `int => ExpressionInterface`.

Reply:
Let's give it a try, I would assume that static analyzers are cool with it, but I'm not sure.

Reply:
Looks like they like it.